unstructured_inference/models/unstructuredmodel.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+from typing import Any
+from PIL import Image
+
+
+class UnstructuredModel(ABC):
+    """Wrapper class for the various models used by unstructured."""
+
+    def __init__(self, model: Any):
```

Comment:
What is model supposed to be, a string, an object, either?
Could use some docstring here.

Reply:
Can this abstract class not bookkeep the model?

Use triton as an example, there interface methods are:

```
   initialize(self, ) [can optionally pass in model_name, config, version, etc]
   execute(self, requests): do an inference for each request
   finalize(self, ): cleanup, "unload"
   auto_complete_config(): the default config
```
   
 Reference:
 https://github.com/triton-inference-server/python_backend/blob/main/examples/pytorch/model.py
 https://github.com/triton-inference-server/python_backend

Reply:
On __init__ vs. initialize. If there is going to be any real work involved, it probably belongs in `initialize()`. I.e., it should be possible to instantiate a class without bogging down your system to do it.

Reply:
Added a docstring.

Reply:
For now, the only work to be done is to download the binary and instantiate the model being wrapped, all of of which is already done by the time this wrapper model is instantiated (since you have to pass it the model binary). We could do it a different way, and include the `load_model` logic in the class itself.